src/__tests__/bundle.spec.ts

```diff
@@ -1619,4 +1620,43 @@ describe('bundleTargetPath()', () => {
       ).not.toThrow();
     });
   });
+
 });
+
+it('whole document refs within a document are broken', () => {
+  const petStore = {
+    openapi: '3.0.0',
+    info: {
+      version: '1.0.0',
+      title: 'Swagger Petstore',
+      description:
+        'A sample API that uses a petstore as an example to demonstrate features in the OpenAPI 3.0 specification',
+      termsOfService: 'http://swagger.io/terms/',
+      contact: {
+        name: 'Swagger API Team',
+        email: 'apiteam@swagger.io',
+        url: 'http://swagger.io',
+      },
+      license: {
+        url: 'https://www.apache.org/licenses/LICENSE-2.0.html',
+        name: 'Apache 2.0',
+      },
+    },
+    servers: [
+      {
+        url: 'http://petstore.swagger.io/api',
+      },
+    ],
+    paths: {
+    },
+    components: {
+      schemas: {
+        Pet: {
+          "$ref": "#",
```

Comment:
This `$ref: #` seems to break everything.  It occurs many times in the PCRecruiter OpenAPI document we were supplied on Wed Feb 8.